renderer/components/Profile/ProfilePaneSignMessage/ProfilePaneSignMessage.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl'
+import { Box, Flex } from 'rebass'
+import { Bar, CopyBox, TextArea, Text, Form, Button } from 'components/UI'
+import messages from './messages'
+
+const ProfilePaneNodeInfo = ({ intl, signMessage, showNotification, ...rest }) => {
+  const [sig, setSig] = useState(null)
+
+  const formApiRef = useRef(null)
+  const notifyOfCopy = () =>
+    showNotification(intl.formatMessage({ ...messages.sig_copied_notification_description }))
+
+  const onSignMessage = async () => {
+    try {
+      const { current: formApi } = formApiRef
+      const message = formApi.getValue('message')
+      const { signature } = await signMessage(message)
+      setSig(signature)
+    } catch {
```

Comment:
I think this is what's causing the test failure @korhaliv - needs to be `catch (e) {`